### PR TITLE
Dmp 3318.duplicate hearing media pk

### PR DESCRIPTION
**Pull request by cmagowan-hmcts**
remove duplicate case numbers before processing addAudio
 https://tools.hmcts.net/jira/browse/DMP-3318

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,3 +30,6 @@ bin/generated-sources/
 
 applicationinsights-agent-*.jar
 *.log
+/src/main/resources/application-perf.yaml
+/src/main/resources/application-dets.yaml
+/src/main/resources/application-demo.yaml
```
Comment (davet1985) [inline .gitignore:35]: What are these for?

Comment (cmagowan-hmcts) [reply .gitignore:35]: these are files i have locally that i don't want to ever accidentally commit, so i am telling git to ignore.

Comment (davet1985) [reply .gitignore:35]: It would be better to add these to your `~/.gitignore` file, so they are only ignored for you.

**Review by chrisbellingham-hmcts — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (cmagowan-hmcts) [reply .gitignore]: removed

**Pull request merged**